elastalert/ruletypes.py

```diff
@@ -431,19 +455,32 @@ def handle_event(self, event, count, qk='all'):
         else:
             self.ref_window_filled_once = True
 
-        if self.find_matches(self.ref_windows[qk].count(), self.cur_windows[qk].count()):
-            # skip over placeholder events which have count=0
-            for match, count in self.cur_windows[qk].data:
-                if count:
-                    break
+        if self.field_value is not None:
+            if self.find_matches(self.ref_windows[qk].mean(), self.cur_windows[qk].mean()):
+                # skip over placeholder events
+                for match, count in self.cur_windows[qk].data:
+                    if "placeholder" not in match:
+                        break
+                self.add_match(match, qk)
```

Comment:
Perhaps we want to add the two means into the match here? Or, don't even use existing data, since cur_windows.data[0] is just a random small piece of data. That last part is slightly confusing but acceptable, we do that in other places. 

Definitely add
```
match['reference_mean'] = self.ref_windows[qk].mean()
match['current_mean'] = self.cur_windows[qk].mean()
```

I think this will be very useful to have.

Reply:
Nevermind this comment. I see it's done in add_match.